CHANGELOG.md

```diff
@@ -59,6 +61,23 @@ introduced in version 0.22.3 and support for binary keys was removed from
 other components in 0.33.0 and 0.37.0. Please migrate to wallets (see 0.37.0
 notes) if you've not done it previously.
 
+To migrate data from Peapods to FSTree:
+```shell
+$ peapod-to-fstree -config </path/to/storage/node/config>
+```
+For any shard, the data from the configured Peapod is copied into
+a FSTree that must be already configured, if it is not, into
+a created FSTree dir named `blobstore` in the directory where the tree is
+located. For example, `/neofs/data/peapod.db` -> `/neofs/data/blobstore`.
+Notice that existing Peapod are untouched. Configuration is also
+updated, for example, `/etc/neofs/config.yaml` -> `/etc/neofs/config_fstree.yaml`.
+WARN: carefully review the updated config before using it in the application!
+
+Now there is an FSTree in which files of any size will be more efficient and easier to store.
+Use only `fstree` sub-storage in `blobstor` config section.
```

Comment:
I'd say that currently we support both, but peapod can be removed in future versions. And we recommend using fstree.